packages/Datepicker/src/Datepicker.stories.tsx

```diff
@@ -131,6 +130,25 @@ export const Disabled: Story = {
   },
 };
 
+export const ReadOnly: Story = {
+  render: Template,
+
+  args: {
+    placeholder: 'Select date',
+    readOnly: true,
+  },
+};
+
+export const French: Story = {
+  render: Template,
+
+  args: {
+    placeholder: 'Select date',
+    locale: 'fr-CA',
+    weekendUnavailable: true
```

Comment:
Wondering what does weekendUnavailable has to do with the French story? This seems to have two concerns bundled into one.

Reply:
I added this here because in the code, it seemed to use locale to calculate the unavailable weekends so I thought it would be good to add to make sure it works properly.

Reply:
I get it, I would split those if it was for visual regression testing, it's ok in this situation.